spec/support/fixtures.rb

```diff
@@ -1630,13 +1638,14 @@ def initialize
 
       @router = Hanami::Router.new do
         get "/",                   to: Mimes::Default.new
-        get "/custom",             to: Mimes::Custom.new
+        # get "/custom",             to: Mimes::Custom.new
         get "/accept",             to: Mimes::Accept.new
-        get "/restricted",         to: Mimes::Restricted.new
+        # get "/restricted",         to: Mimes::Restricted.new
         get "/latin",              to: Mimes::Latin.new
         get "/nocontent",          to: Mimes::NoContent.new
         get "/overwritten_format", to: Mimes::OverrideDefaultResponse.new
-        get "/custom_from_accept", to: Mimes::CustomFromAccept.new
+        # get "/custom_from_accept", to: Mimes::CustomFromAccept.new
+        get "/strict",             to: Mimes::Strict.new
```

Comment:
@timriley - all the examples in the `mime_type_spec.rb` file were `xit`. In getting that spec to execute the new examples I added to it, I had to remove these particular fixture actions to avoid the below error being hit when running my new examples.

I think perhaps however the controller used to be configured with custom formats is now different or unsupported?

```
Failures:

  1) MIME Type Content type with an accepted format does not accept an unmatched format
     Failure/Error: TYPES.fetch(format) { raise Hanami::Controller::UnknownFormatError.new(format) }
     
     Hanami::Controller::UnknownFormatError:
       Cannot find a corresponding Mime type for 'custom'. Please configure it with Hanami::Controller::Configuration#format.
     # ./lib/hanami/action/mime.rb:119:in `block in format_to_mime_type'
     # ./lib/hanami/action/mime.rb:119:in `fetch'
     # ./lib/hanami/action/mime.rb:119:in `format_to_mime_type'
     # ./lib/hanami/action/mime.rb:151:in `block in restrict_mime_types'
 ```